package/src/store/QuickSqliteClient.ts

```diff
@@ -123,26 +105,24 @@ export class QuickSqliteClient {
   };
 
   static initializeDatabase = () => {
-    // @ts-ignore
     if (sqlite === undefined) {
       throw new Error(
         'Please install "react-native-quick-sqlite" package to enable offline support',
       );
     }
 
-    if (this.isQuickSqliteV4) {
-      console.warn(
-        'You seem to be using an older version of "react-native-quick-sqlite" dependency,',
-        'and we are going to drop support for it in the next major release.',
-        'Please upgrade to the version v5 of "react-native-quick-sqlite" to avoid any issues.',
+    if (QuickSqliteClient.isQuickSqliteV4) {
+      console.error(
+        'You seem to be using an older version of "react-native-quick-sqlite" dependency.',
+        'Please upgrade to the version v5 (or higher) of "react-native-quick-sqlite" to avoid any issues.',
```

Comment:
instead I would suggest upgrading peer dependency to >= v5 and drop any references of v4. Much more easier.